src/components/PostCommentForm.jsx

```diff
@@ -0,0 +1,46 @@
+import { addCommentByArticleId } from "../../api";
+import { useState } from "react";
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
+
+function PostCommentForm({ id, setComments }) {
+  const [newComment, setNewComment] = useState("");
+  const { user } = useContext(UserContext);
+
+  function handleSubmit(event) {
+    addCommentByArticleId(id, newComment, user)
+      .then((response) => {
+        alert(`You posted your comment!`);
+        setComments((currComments) => {
+          return [response.data.comment, ...currComments];
+        });
+      })
+      .catch((error) => {
+        alert(`Oopsie, there's been an error. Try again please.`);
```

Comment:
Would recommend looking into how we can include error feedback in the design of our ui rather than alerts - a lot of users have pop-up blockers nowadays so they're rarely the first choice for feeding back on errors/success/loading states and are not necessarily the most accessible choice.